apollo-normalized-cache-sqlite/src/commonTest/kotlin/com/apollographql/apollo/cache/normalized/sql/SqlNormalizedCacheTest.kt

```diff
@@ -54,7 +54,19 @@ class SqlNormalizedCacheTest {
   }
 
   @Test
-  fun testRecordMerge() {
+  fun testRecordMerge_noOldRecord() {
```

Comment:
I think you missing `createRecord(STANDARD_KEY)` before `cache.merge(`

Reply:
I've split the test in 2 for when a record is already present or not to test both branches